src/jit/gentree.cpp

```diff
@@ -7817,6 +7817,504 @@ GenTreePtr GenTree::GetChild(unsigned childNum)
     }
 }
 
+GenTreeOperandIterator::GenTreeOperandIterator()
+    : m_node(nullptr)
+    , m_operand(nullptr)
+    , m_argList(nullptr)
+    , m_multiRegArg(nullptr)
+    , m_expandMultiRegArgs(false)
+    , m_state(-1)
+{
+}
+
+GenTreeOperandIterator::GenTreeOperandIterator(GenTree* node, bool expandMultiRegArgs)
+    : m_node(node)
+    , m_operand(nullptr)
+    , m_argList(nullptr)
+    , m_multiRegArg(nullptr)
```

Comment:
>  But I'm guessing how you have it is consistent with the rest of our code and/or the style guide,

You are correct: https://github.com/dotnet/coreclr/blob/master/Documentation/coding-guidelines/clr-jit-coding-conventions.md#951-constructor-with-member-initialization-list